docs/VideoPlayer.md

```diff
@@ -0,0 +1,102 @@
+# Video Player
+
+The VideoPlayer component allows you to stream a video from the Kinesis Video Streams. It supports the following configuration:
```

Comment:
is there more detail we could provide? My understanding is it's more than _just_ a video player, but more like a playlist?

Reply:
I think the details added are sufficient and clear for the users to understand the purpose and usage of the video player component.
From user point of view it is regular video player that can play from KVS using KVS stream name or using TwinMaker Entity and Component information.
Internally, in the TwinMaker mode we fetch the KVS stream name and other information like when the video is available, when not available and when available on edge. And then accordingly setup the video player with sources. I think we should not include these information in the user guide for the component. 

I'm open to recommendations though - Please suggest.